// Vortex messaging protoype
// VERY Basic protocol

message sensor_data {
    repeated uint32 ir_data_array = 1 [packed=true];
    repeated uint32 sonar_data_array = 2 [packed=true];
    optional uint32 timestamp = 3;
}


message command_data{
  enum  CommandType{
    MOVE = 0;
    TURN = 1;
    STOP = 2;
  }

  optional CommandType command = 1 [default = STOP];
  repeated int32 args = 2 [packed = true];
}
